app/services/marketing_site.rb

```diff
@@ -61,14 +61,16 @@ def self.security_url
   end
 
   def self.help_center_article_url(category:, article:, article_anchor: '')
-    if !valid_help_center_article?(category:, article:, article_anchor:)
-      raise ArgumentError.new("Unknown help center article category #{category}/#{article}")
+    if !HELP_CENTER_ARTICLES.include?("#{category}/#{article}")
+      raise ArgumentError, "Unknown help center article category #{category}/#{article}"
     end
     anchor_text = article_anchor.present? ? "##{article_anchor}" : ''
     URI.join(BASE_URL, locale_segment, "help/#{category}/#{article}/#{anchor_text}").to_s
   end
 
-  def self.valid_help_center_article?(category:, article:, **_article_anchor)
-    HELP_CENTER_ARTICLES.include?("#{category}/#{article}")
+  def self.valid_help_center_article?(category:, article:, article_anchor: '')
+    !!help_center_article_url(category:, article:, article_anchor:)
+  rescue URI::InvalidURIError, ArgumentError
```

Comment:
I do not love rescuing `ArgumentError` here. I would almost rather create a `MarketingSite::UnknownArticleException` so we can rescue that.